packages/public/common/src/hooks/useSightLabel.ts

```diff
@@ -0,0 +1,31 @@
+import { LabelDictionary, Sight } from '@monkvision/types';
+import { useObjectTranslation } from './useObjectTranslation';
+
+/**
+ * The result of the useSightLabel. It contains a function which takes a Sight object and return the
+ * translated label sync with the actual selected language.
+ */
+export interface UseSightLabelResult {
+  label: (sight: Sight) => string;
+}
+
+export interface UseSightLabelParams {
+  /**
+   * Function translating a Sight object into a translated label sync with the actual selected language.
+   * @param obj
```

Comment:
Update the TSDoc for this field (it is not a function), and remove the @param annotation